docs/overrides/home.html

```diff
@@ -38,13 +34,11 @@
                         </div>
                     </div>
                 </div>
-                
                 <div class="relative px-20 items-center gap-12 m-auto lg:inline-flex md:order-first">
-                    <a href="https://form.jotform.com/231974699426069" target="_blank">
-                        <img src="assets/img/logo_transp.png" alt="Help us find a logo"
-                                class="mx-auto w-[24rem] my-2 rounded-xl shadow-xl ring-1 ring-gray-400/10 sm:w-[40rem]"
-                                width="400" height="400">
-                    </a>
+                    <!-- TODO: replace with new logo when ready. -->
```

Comment:
Cool!

In the meantime, what do you think about centering the corresponding hero section ("An ecosystem for distributed dataset harmonization and search. ...") since the left-justification currently looks a little odd on the site?

We could comment out the existing `div` that defines the left-align so we can easily bring it back once the logo is ready.

Reply:
sure, sounds good. I don't know what div you want to comment out though - I don't see a left-align here. Want to just make a commit to the PR branch?

Reply:
@surchs done - feel free to merge if it looks good!